pilz_teleoperation/test/unittests/tst_ros_message_serializer.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Pilz GmbH & Co. KG
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import rospy
+import pytest
+from pilz_teleoperation.ros_message_serializer import RosMessageSerializer
```

Comment:
its in __init__.py so module does not have to be named
```suggestion
from pilz_teleoperation import RosMessageSerializer
```